torchtext/experimental/vectors.py

```diff
@@ -149,18 +83,19 @@ def GloVe(name="840B", dim=300, unk_tensor=None, root=".data", validate_file=Tru
         root (str): folder used to store downloaded files in (.data)
         validate_file (bool): flag to determine whether to validate the downloaded files checksum.
                               Should be `False` when running tests with a local asset.
+        num_cpus (int): the number of cpus to use when loading the vectors from file. Default: 10.
     Returns:
         Vectors: a Vectors object.
 
     Raises:
         ValueError: if unexpected duplicate tokens are found in GloVe file.
 
     """
-    dup_token_glove_840b = [("����������������������������������������������������������������������"
-                             "����������������������������������������������������������������������"
-                             "����������������������������������������������������������������������"
-                             "����������������������������������������������������������������������"
-                             "������������������������������������������������������", 140649)]
```

Comment:
What happened here?

Reply:
I believe we discussed the fact that torchbind doesn't allow inner tuple return types which is why I had to remove the tuple of (dup_tokens, line_number). If you still think this is very important, I could return 2 separate `std::vectors`, one containing dup_tokens and the other containing dup line numbers?

Reply:
This change seems to remove the ability to also check the line number the duplicate was found on. That's an important check, so whatever need to do to maintain that seems important.